src/schema/v2/__tests__/user.test.ts

```diff
@@ -497,7 +497,7 @@ describe("User", () => {
         {
           owner_type: "ArtworkInquiryRequest",
           sale_price: 600.1234,
-          sale_date: "2018-09-26 04:00:00 UTC",
+          sale_date: "Sep 26th, 2018",
```

Comment:
I am curious why the test returns `invalid date` here when they seemed to have had the same string before 🤔

Reply:
Perhaps this is because the stub has changed. It went from `sale_date: "2018-09-26 04:00:00 UTC"` to `sale_date: "Sep 26th, 2018",`.

The block defined starting [L:477](https://github.com/artsy/metaphysics/blob/kajatiger/PLATFORM-4793-format-sale_date-string-to-be-date-in-metaphysics/src/schema/v2/__tests__/user.test.ts#L477-L517) `purchases` is the stubbed shape coming in from gravity.

I think you changed the format in the wrong place. The date as stubbed to be coming from gravity doesn't change, but the expected shape after it has gone through metaphysics should have changed now that the query specifies a format.

So [L:547](https://github.com/artsy/metaphysics/pull/4612/files#diff-62c68948ab97aaf79e25a22d7344f59bd6ff8bb78af7fa6fe9ef020920408d5dR547) should probably be `Sep 26th, 2018`

Reply:
Ah okay, makes sense! Thanks @ovasdi !!